Exam/CT/src/exercice1/MathTest.java

```diff
@@ -0,0 +1,43 @@
+package exercice1;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import org.junit.jupiter.api.Test;
+
+class MathTest {
+
+	@Test
+	void test() {
+    	// Cr�ation de la matrice d'exemple
+        int[][] matrice = new int[5][5];
+        for(int i = 0; i < matrice.length; i++){
```

Comment:
Hmmmm... Ok, but note that int[][] object are filled with 0 by default when initialized